CONTRIBUTING.md

```diff
@@ -0,0 +1,132 @@
+# Contributing to the OpenAPI Specification
+
+***Work in progress!***
+
+We are currently working on [defining and documenting our new processes](https://github.com/orgs/OAI/projects/5).  Information in this document is up-to-date.  Older _(and sometimes now inaccurate)_ documentation can be found in [DEVELOPMENT.md](DEVELOPMENT.md), which will be removed when everything is updated and documented here.
+
+## Essential Policies
+
+This section serves as a quick guide while we work on the full updated documentation.
+
+If in doubt about a policy, please [ask on our Slack](https://communityinviter.com/apps/open-api/openapi) before opening a PR.
+
+### No changes to published specifications
+
+No changes, ***no matter how trivial***, are ever made to the contents of published specifications.  The only potential changes to those documents are updates to link URLs _if and only if_ the targeted document is moved by a 3rd party.  Other changes to link URLs are not allowed.
+
+### Current branches and documents open to change
+
+The first PR for a change should be against the oldest release line to which it applies.  Changes can then be forward-ported as appropriate.
+
+The current (31 July 2024) active releases are:
+
+| Version | Branch | File | Notes |
+| ------- | ------ | ---- | ----- |
+| 3.0.4 | `v3.0.4-dev` | `versions/3.0.4.md` | Soon to be published |
+| 3.1.1 | `v3.1.1-dev` | `versions/3.1.1.md` | Soon to be published |
+| 3.2.0 | `v3.2.0-dev` | `versions/3.2.0.md` | Planned for late 2024 |
+| 4.0.0 | [OAI/sig-moonwalk](https://github.com/OAI/sig-moonwalk) | [discussions only](https://github.com/OAI/sig-moonwalk/discussions) | TBD, some proposals to be backported and published in 3.2.0 |
+
```

Comment:
I think we need a section on "Markdown style". We have a set of conventions for how we write the spec documents -- we should capture these here and say that we enforce some of them with markdown lint (if we do). I think it would also be good to document our conventions for RFC and other links.

Reply:
@mikekistler this is just to start an outline and populate it with material directly copied over - for additional content, we need to do additoinal PRs.